### PR TITLE
Add all DataKit 0.9.0 packages

### PR DESCRIPTION
**Pull request by samoht**
This includes the new DataKit CI package.

See https://github.com/docker/datakit/releases/tag/0.9.0 for more details

### CODE REVIEW

File: packages/datakit-ci/datakit-ci.0.9.0/opam [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  "tls"
+  "channel"
+  "conduit"
+  "io-page"
+  "pbkdf"
+  "webmachine" {>= "0.3.2"}
+  "session" {>= "0.3.0"}
+  "redis"
+  "asetmap"
+  "github"
```
Comment (avsm) [inline packages/datakit-ci/datakit-ci.0.9.0/opam]: github needs a lower bound of `github >= 2.2.0` i think

File: packages/datakit-client/datakit-client.0.9.0/descr [hunk 2]
```diff
@@ -0,0 +1,5 @@
+A library to connect to datakit servers
```
Comment (avsm) [inline packages/datakit-client/datakit-client.0.9.0/descr]: s/datakit/DataKit/ to be consistent

File: packages/datakit-client/datakit-client.0.9.0/opam [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+build: [
+  "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%" "-n" name
+]
+
+depends: [
+  "ocamlfind"  {build}
+  "ocamlbuild" {build}
+  "topkg"      {build}
+  "base-bytes"
+  "astring" "logs" "uri" "rresult" "cstruct" "fmt"
```
Comment (avsm) [inline packages/datakit-client/datakit-client.0.9.0/opam]: cstruct needs a lower bound of `>= 2.2.0` i think

File: packages/datakit-server/datakit-server.0.9.0/opam [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+]
+
+depends: [
+  "ocamlfind"  {build}
+  "ocamlbuild" {build}
+  "topkg"      {build}
+  "base-bytes"
+  "astring" "logs" "uri" "rresult" "cstruct" "fmt"
+  "protocol-9p" {>= "0.7.4"}
+  "sexplib" "mirage-types-lwt"
```
Comment (avsm) [inline packages/datakit-server/datakit-server.0.9.0/opam]: `mirage-types-lwt` needs to be `<3.0.0`

File: packages/datakit-server/datakit-server.0.9.0/opam [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+build: [
+  "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%" "-n" name
+]
+
+depends: [
+  "ocamlfind"  {build}
+  "ocamlbuild" {build}
+  "topkg"      {build}
+  "base-bytes"
+  "astring" "logs" "uri" "rresult" "cstruct" "fmt"
```
Comment (avsm) [inline packages/datakit-server/datakit-server.0.9.0/opam]: `cstruct >= 2.2.0`

**Comment by samoht** [comment]
Updated

**Comment by samoht** [comment]
Travis is timing out on macOS but is green otherwise. @avsm are you happy to merge? The other errors seem due to the fact that not all packages in that PR were pinned when running the revdep tests?

**Comment by talex5** [comment]
The first failure I see is:
```
- ocamlfind: Package `hvsock.lwt-unix' not found
- Command exited with code 2.
- pkg.ml: [ERROR] cmd ['ocamlbuild' '-use-ocamlfind' '-classic-display' '-tag' 'debug'
-      '-build-dir' '_build' '-Is' 'src,bridge/github' '-package'
-      'datakit-client,datakit-server.vfs,datakit-server.fs9p,datakit-github'
-      'CHANGES.md' 'LICENSE.md' 'README.md' 'bridge/github/main.native']: exited with 10
[ERROR] The compilation of datakit-bridge-github failed at "ocaml pkg/pkg.ml build --pinned false -n datakit-bridge-github --tests false".
```

**Comment by samoht** [comment]
Constraints to hvsock added.

**Comment by talex5** [comment]
CI failed with:
```
opam: "mkdir" failed on /home/opam/.opam/packages.dev/ppx_core.113.33.03: No space left on device
```
and
```
Uncaught exception: Fl_topo.Inconsistent_ordering
```

**Comment by samoht** [comment]
The 1st issue is an infrastructure pb, the 2nd one is a findlib bug :-/

I think we are good to merge. Just waiting for @avsm to validate the changes.

**Comment by avsm** [comment]
sorry about the delay, merging away!

**Pull request merged**